src/log-component/providers/README.md

```diff
@@ -0,0 +1,103 @@
+# Providers
+
+This directory contains providers contributing additional bindings, for example
+custom sequence actions.
+
+## Overview
+
+A provider is a class that provides a `value()` function. This function is called `Context` when another entity requests a value to be injected.
+
+Here we create a provider for a logging function that can be used in a custom sequence action.
+
+The logger will log the URL, the parsed request parameters, and the result. The logger is also capable of timing the sequence if you start a timer at the start of the sequence using `process.hrtime()`.
+
+## Basic Usage
+
+### LogProvider
+
+LogProvider can automatically be bound to your Application's Context using the LogComponent which exports the provider on with a binding key of `extension-starter.actions.log`. You can learn more about components in the reference section. 
```

Comment:
- Break this up into separate lines
- `...provider on with a binding key...` should probably say `...provider with a binding key...`
- `You can more about components in the reference section` --> is this meant to be a link?